helm/values.yaml

```diff
@@ -190,6 +190,8 @@ llama-stack:
         actions: [read, update, delete]
       when: user is owner
       description: users can read, update and delete resources they own
+  secrets:
+    TAVILY_SEARCH_API_KEY: "api-key"
 
```

Comment:
may be leaving empty might be a good rather than having api-key, it may be hard to debug unless we run for it. and also this gives me suspicion that if application is not smart enough to display error message if the `TAVILY_SEARCH_API_KEY` is wrong. what would happen in that scenario?

Reply:
Done